File: frontEnd/src/app/types/cv.ts
export class CV {
  public email: string;
  public filename: string;
  public uploaded: string;
  public file: string;
  public type: string;
  public size: number;

  constructor(
    email: string,
    filename: string,
    uploaded: string,
    file: string,
    type: string,
    size: number,
  ) {
    this.email = email;
    this.filename = filename;
    this.uploaded = uploaded;
    this.file = file;
    this.type = type;
    this.size = size;
  }
}

File: backend/BackendService/Model/CV.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace BackendService.Model
{
    [Table("CV")]
    public class CV
    {
        [Key]
        public int? Id { get; set; }
        public string Email { get; set; }
        public string Filename { get; set; }
        public string File { get; set; }
        public string Type { get; set; }
        public DateTime Uploaded { get; set; }
        public int Size { get; set; }
    }
}
